app/backend/Services/AzureComputerVisionService.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net.Http.Headers;
+using System.Text;
+
+namespace MinimalApi.Services;
+
+public class AzureComputerVisionService(IHttpClientFactory httpClientFactory, string endPoint, string apiKey)
```

Comment:
There seems to be a lot of repeated code in here. I really think using an SDK would simplify this, and help to DRY it up.

Reply:
The link you shared above is for image analysis, while the api we want to use here is image retrieval

We can always clean up the code once sdk for image retrieval is available. The clean up process would also be easy as the change would be limited to a single class only